mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,114 @@
 @import "src/styles/variables";
 
+$maxWidth: 1200px;
+
 .pageWrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 0 $gapMedium;
   margin-top: $marginMedium;
+  gap: $gapBig;
+}
+
+.title {
+  color: var(--secondaryTextButtonColor);
+  text-align: center;
+}
+
+.subtitle {
+  color: var(--primaryTextColor);
+  text-align: center;
+  margin-bottom: $marginSmall;
+}
+
+.aboutBlock {
+  max-width: $maxWidth;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $gapSmall;
+  margin-top: $marginBig;
+
+  color: var(--secondaryTextColor);
+  background-color: var(primarysBackgroundColor);
+}
+
+.aboutDescription {
+  display: flex;
+  align-items: center;
+  margin-top: $marginBig;
+  background-color: var(--primaryBackgroundColor);
+}
+
+.aboutTextBlock {
+  flex: 1 1 70%;
+  padding: $paddingMedium;
+  display: flex;
+  flex-direction: column;
+  gap: $gapBig;
+  line-height: 1.5;
+  color: var(--primaryTextColor);
+  background-color: var(--primaryBackgroundColor);
+}
+
+.logoAbout {
+  flex: 1 1 30%;
+  padding: $paddingMedium;
+  background-color: var(--primaryBackgroundColor);
+  @media (max-width: 850px) {
+    display: none;
+  }
+}
+
+.ourTeamBlock {
+  color: var(--secondaryTextColor);
+  background-color: var(--tertiaryBackgroundColor);
+  width: 100%;
+  padding: $paddingBig;
+  display: flex;
+  flex-direction: column;
+  gap: $gapLarge;
+  text-align: center;
+}
+
+.ourTeamMembers {
+  display: flex;
+  justify-content: center;
+  gap: $gapLarge;
+  overflow-x: auto;
+  &::-webkit-scrollbar {
+    height: 8px;
+    background-color: var(--tertiaryBackgroundColor);
+    border-radius: 10px
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: var(--secondaryTextColor);
+    height: 8px;
+    border-radius: 10px;
+  }
 }
 
-.projectDescription {
-  max-width: 600px;
+.ourTeamMember {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: $gapMedium;
+}
+
+.avatar {
+  margin-bottom: $marginMedium;
 }
 
 .accordion {
+  background-color: var(--primaryBackgroundColor);
+  box-shadow: none;
+}
+
+.accordionSection {
+  gap: $gapBig;
+  max-width: $maxWidth;
   width: 100%;
-  max-width: 600px;
+  line-height: 1.3;
```

Comment:
why do we need to change line-height?

Reply:
it's better with line-height: 1.3; 
Without line-height 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/d366d658-9e01-4993-8c13-d1ce7a5923cd)
with line-height 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/d380d941-adbb-4c98-ae52-8e1fe608caca)

Reply:
is it common problem? why does it related to this component only?

Reply:
I talked to Victoria about the line-height, she said it needed, and she will add them to the Figma maket. Currently, in the Figma maket line-height by default has auto. I'll remove it temporarily

Reply:
maybe the radix component have another default values